docs/functions/data.rst

```diff
@@ -52,7 +52,7 @@ Development Environment
    Returns ``true`` if a Kubernetes context is found.
    Returns the Kubernetes context name or the first name component.
 
-   Splitting long context names into components is defined by :attr:`LP_DELIMITER_KUBECONTEXT`.
+   Splitting long context names into components is defined by :attr:`LP_DELIMITER_KUBECONTEXT` and :attr:`LP_DELIMITER_KUBECONTEXT_STRIP_PREFIX.`
```

Comment:
The changes on this file duplicate the changes you added in 545d297, which conflict.

Reply:
I believe this is resolved as of today's rebase.